docs/DeveloperGuide.adoc

```diff
@@ -734,16 +776,138 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
+
+[discrete]
+=== Use case: Add person
+
+*MSS*
+
+1.  User requests to add person with the person’s name, phone number and email address
+2.  AddressBook adds the person
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. User did not give enough details
++
+AddressBook displays an error message
++
+Use case ends.
+
+
+[discrete]
+=== Use case: View command history
+
+*MSS*
+
+1.  User requests to view command history
+2.  AddressBook displays the command history in reverse chronological order
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. AddressBook displays an empty list
```

Comment:
this should be 1a. as it prevents you from going to 2.
Also, it should be `command history is empty`